README.md

```diff
@@ -134,7 +134,11 @@ Read more about [Netlify redirects](https://docs.netlify.com/routing/redirects/)
 
 ### Versions
 
-You can check our `package.json` for supported Next and Node versions. Our support of Next 10 is currently experimental.
+You can check our `package.json` for supported Node versions.
+
+For breaking Next.js releases, all Next.js versions before the breaking release will be locked to the last working plugin version:
+
+Next.js < 10.0.6 is locked to plugin version 1.1.3. To use the latest plugin version, upgrade to Next.js >= 10.0.6.
```

Comment:
Should we replace `1.1.3` by `1.*.*` or `v1` to avoid updating the `README.md` when new backports are made to `v1`?

Reply:
how often are new backports gonna be made? i feel like it should be rare enough that it's not a big deal, people should probably just be upgrading

Reply:
:+1: